### ISSUE TITLE
M2 Air - texts barely visible and all messed up

### ISSUE THREAD

**Issue by harryqt** [imfile-io/imfile-desktop#183]
<img width="1105" alt="Screenshot 2024-11-22 at 2 29 33 PM" src="https://github.com/user-attachments/assets/59979644-a5ec-4071-8cb2-6c65ee4bceef">
<img width="1105" alt="Screenshot 2024-11-22 at 2 29 20 PM" src="https://github.com/user-attachments/assets/4604d09b-7b11-4119-9d37-7a2e40756c72">

**Comment by LampPrinter** [comment]
Same.

**Comment by lucky13820** [comment]
Yes, I switched back to Motrix because of this reason. It's barely usable.

**Comment by smounives** [comment]
Windows version has the same problem
![image](https://github.com/user-attachments/assets/7938f9e2-f987-4508-b16e-6833f03bd9f3)

**Comment by sukibaby** [comment]
Fixed in #193
